contents/docs/self-host/deploy/configuration.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Posthog Clickhouse Chart Configuration
+sidebarTitle: Configuration
+sidebar: Docs
+showTitle: true
+---
+
+## Dependencies
+
+By default, the chart installs the following dependencies:
+
+- [Altinity/clickhouse-operator](https://github.com/Altinity/clickhouse-operator/)
+- [bitnami/postgresql](https://github.com/bitnami/charts/tree/master/bitnami/postgresql)
+- [bitnami/redis](https://github.com/bitnami/charts/tree/master/bitnami/redis)
+- [bitnami/kafka](https://github.com/bitnami/charts/tree/master/bitnami/kafka)
+
+There is optional support for the following additional dependencies:
+
+- [kubernetes/ingress-nginx](https://github.com/kubernetes/ingress-nginx/)
+- [jetstack/cert-manager](https://github.com/jetstack/cert-manager)
+- [prometheus-community/prometheus](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus)
+- [prometheus-community/prometheus-statsd-exporter](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-statsd-exporter)
+
+See the [Configuration](#configuration) section for how to enable and configure each.
+
+## Configuration
+
+Dependent charts can also have values overwritten. Preface values with `postgresql.*`, see options for each chart in [ALL_VALUES.md](ALL_VALUES.md) or under each charts repos in [Dependencies](#dependencies).
+
+All configuration options can be found in [ALL_VALUES.md](ALL_VALUES.md) or in [values.yaml](values.yaml) file.
+
+### Setting up email
+Outgoing email is used for password reset for example. For posthog to be able to send emails we need a login and password. Add these settings to your `values.yaml`:
+```yaml
+email:
+  user: <your STMP login user>
+  password:  <your STMP password>
+```
+
+### Scaling up
+The default configuration is geared towards minimizing costs. Here are example extra values overrides to use for scaling up:
```

Comment:
This does feel like it should be in the top-level Configure. Likely not a blocker.

Reply:
Agreed, will move later.